.github/workflows/runner.yml

```diff
@@ -25,18 +25,8 @@ jobs:
         cat /proc/self/status
         systemd-detect-virt
     - name: Install dependencies
-      run: |
-        sudo cat /etc/apt/sources.list
-        sudo sed -i '/deb-src/d' /etc/apt/sources.list
-        sudo sed -i '/^deb /p;s/ /-src /' /etc/apt/sources.list
-        export DEBIAN_PRIORITY=critical
-        export DEBIAN_FRONTEND=noninteractive
-        # let's try to work around upgrade breakage in a pkg we don't care about
-        sudo apt-mark hold grub-efi-amd64-bin grub-efi-amd64-signed
-        sudo apt-get update
-        sudo apt-get -y dist-upgrade
-        sudo apt-get -y install ubuntu-dev-tools automake autopoint xsltproc gettext expect byacc libtool libbsd-dev pkgconf libcmocka-dev
-        sudo apt-get -y build-dep shadow
+      id: dependencies
+      uses: ./.github/actions/install-dependencies
```

Comment:
For patch 2, 5ebb973150e383c835a92b107bed1cb800feadb4 ("CI: run Install dependencies workflow "):
```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```

Reply:
Done